config/development.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
it should not be here

Reply:
add this file to .gitignore on your machine

Reply:
.gitignore is common for all(because .gitignore is not in .gitignore).But the file was added to the пше before in .gitignore. I will remove it from git cache